src/pkg/project/dao/dao.go

```diff
@@ -199,3 +201,20 @@ func (d *dao) ListRoles(ctx context.Context, projectID int64, userID int, groupI
 
 	return roles, nil
 }
+
+func (d *dao) ListAdminRolesOfUser(ctx context.Context, userID int) ([]models.Member, error) {
+	o, err := orm.FromContext(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	sql := `select b.* from project as a left join project_member as b on a.project_id = b.project_id where a.deleted = 'f' and b.entity_id = ? and b.entity_type = 'u' and b.role = 1;`
```

Comment:
With the LDAP_AUTH and OIDC_AUTH, what if the current user is a group member and the group is granted to be a member with the admin role?

Reply:
I will raise another PR to resolve it, thanks.

Reply:
We ran into this exact issue with version 2.10.0. Users are members of Groups and those Groups are Project Admin members. This worked in 2.9.1. Is there a PR already opened?